files/sandbox-image.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Pulls the containerd sandbox image periodically
+
+[Timer]
+OnUnitActiveSec=60
```

Comment:
This needs to happen more frequently than kubelet's image GC, which is every 2 minutes. We don't want it to happen *too* frequently though, to stay within ECR API limits if this issue happens on many nodes at once.